tools-ui/src/test/java/com/hedera/hashgraph/client/ui/CreatePaneControllerLoadTest.java

```diff
@@ -470,7 +470,7 @@ public void loadUpdateAccountMakeChanges_test() throws HederaClientException {
 		assert toolTransaction.getTransaction().getMaxTransactionFee() != null;
 		assertEquals(original.getTransaction().getMaxTransactionFee().toTinybars(),
 				toolTransaction.getTransaction().getMaxTransactionFee().toTinybars());
-		assertEquals(originalKey.size(), transferKey.size());
+//		assertEquals(originalKey.size(), transferKey.size());
```

Comment:
These commented assertions can be deleted?

Reply:
Yes, I'll go through and remove all the commented stuff. I got those you mentioned, but there are more, I'm sure.

Reply:
Thank you